src/librustdoc/formats/renderer.rs

```diff
@@ -102,5 +96,5 @@ crate fn run_format<'tcx, T: FormatRenderer<'tcx>>(
         }
     }
     prof.extra_verbose_generic_activity("renderer_after_krate", T::descr())
-        .run(|| format_renderer.after_krate(&krate, diag))
+        .run(|| format_renderer.after_krate(crate_name, diag))
```

Comment:
It took me a second to realize: the reason this changed is because you partially moved out of `krate` when adding `krate.module` to the work list.

Reply:
Yeah, I mentioned that in the PR description:

> I'm wondering if it was originally there so that we could take the
module which enables after_krate to take an &Crate. However, the two
impls of after_krate only use Crate.name, so we can pass just the
name instead.

Reply:
Right - that explains that it's possible, the `krate.module` thing explains why it's necessary. :)

Reply:
Yeah, I should've included more information :)